### DISCUSSION TITLE
Welcome to tdesign-vue-next Discussions!

### DISCUSSION THREAD

**Discussion by PengYYYYY** [Tencent/tdesign-vue-next#65] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kitorv** [comment]
使用<script setup>还是普通的<script>方式进行开发？

**Reply by PengYYYYY** [reply]
组件使用tsx开发，demo示例现在使用<script>开发的，<script setup>与<script>都可以

**Comment by daixiangtian** [comment]
tdesign-vue-next 样式打包有问题。本地开发，样式加载顺序是正常的，但是打包之后样式顺序有错乱。t-button
本地环境
![image](https://user-images.githubusercontent.com/45963820/167835917-90ce2f69-6979-4070-a5ab-2aae305a0e36.png)
生成环境
![image](https://user-images.githubusercontent.com/45963820/167836106-2df02250-adde-41de-9ec7-01614031bf5c.png)

**Reply by PengYYYYY** [reply]
> tdesign-vue-next 样式打包有问题。本地开发，样式加载顺序是正常的，但是打包之后样式顺序有错乱。t-button 本地环境 ![image](https://user-images.githubusercontent.com/45963820/167835917-90ce2f69-6979-4070-a5ab-2aae305a0e36.png) 生成环境 ![image](https://user-images.githubusercontent.com/45963820/167836106-2df02250-adde-41de-9ec7-01614031bf5c.png)

新版本是否还存在这样的情况呢？

**Comment by ldsenow** [comment]
Wondering if we can have English translation in the docs so our team mates that arent Chinese background can use.

**Reply by PengYYYYY** [reply]
> Wondering if we can have English translation in the docs so our team mates that arent Chinese background can use.

TDeisgn is a relatively large design system, the translation of documents will be a special work, and you can follow our internationalization special. Thank you for your interest in TDesign. We are also sorry to reply to your message only now!
